mcs/class/corlib/System/Console.cs

```diff
@@ -97,7 +97,6 @@ public static void RemoveCtrlHandler ()
 		static Console ()
 		{
 			if (Environment.IsRunningOnWindows) {
-#if !MOBILE				
```

Comment:
this needs to be FEATURE_CONSOLE as well or it'll break on monotouch because of missing WindowsConsole.